src/main/resources/assets/betterfps/lang/zh_cn.lang

```diff
@@ -0,0 +1,98 @@
+# Simplified Chinese Contributors
+# Degh_Cilon_ - You can't connect me!:-)
+#Some jokes mentioned and their sources are listed below:
+#betterfps.update.phrase.3 - Undertale
+#betterfps.update.phrase.4 - The Stanley Parable
+#betterfps.update.phrase.10 - "bugjump"
+
+betterfps.update.available=BetterFps %s 现已可用
+betterfps.update.prompt=下载新版本？
+betterfps.update.button.download=[下载]
+betterfps.update.button.download.info=点此下载新版本
+betterfps.update.button.more=[更多]
+betterfps.update.button.more.info=点此获取更多版本
+
+betterfps.update.phrase.1=只是另一个烦人的更新。
+betterfps.update.phrase.2=§m在此打广告
+betterfps.update.phrase.3=见到一个新的更新推出，这使你充满了决心。
+betterfps.update.phrase.4=当Stanley走进更新提醒，他按下了“下载”按钮。
+betterfps.update.phrase.5=不会这样了D:
+betterfps.update.phrase.6=<3
+betterfps.update.phrase.7=我知道你会更新的，仅仅为了摆脱这个提醒
+betterfps.update.phrase.8=至少，这不收费...
+betterfps.update.phrase.9=另一个烦人的更新检测器？(╯°□°）╯︵ §m┻━━┻
+betterfps.update.phrase.10=一些漏洞被修复了，但是一些新的被引进了 ˉ\_(ツ)_/ˉ
+betterfps.update.phrase.11=因为自动更新可能会把你的游戏弄得乱七八糟
+betterfps.update.phrase.12=你想来点新东西吗？在这里，有一个更新。
+
+betterfps.installer.title=BetterFps安装器
+betterfps.installer.versions=版本%s 适用于 Minecraft %s
+betterfps.installer.note.1=此安装器为使用官方启动器的原版Minecraft而创建。
+betterfps.installer.note.2=如果你正在使用Forge或LiteLoader，将本文件提取至mods文件夹里。
+betterfps.installer.note.3=启动器或许需要关闭或重启。
+betterfps.installer.button.install=安装
+betterfps.installer.button.extract=提取
+betterfps.installer.button.algorithm=算法测试
+betterfps.installer.button.downloads=下载
+betterfps.installer.button.issues=漏洞/建议
+betterfps.installer.label.game-dir=Minecraft安装目录
+betterfps.installer.label.version=版本
+betterfps.installer.label.profile=简况
```

Comment:
Sure, feel free to ask me anything, and when you're done, just tell me and I'll merge :)